samples/create_event.cc

```diff
@@ -1,8 +1,127 @@
 #include "proto/spec.pb.h"
+#include <fstream>
 #include <iostream>
-int main() {
+
+void GetUserInput(std::string prompt, std::string& res) {
+    std::cout << prompt << ": ";
+    getline(std::cin, res);
+}
+
+void OutputToInterface(std::string content, int type) {
+    switch(type) {
```

Comment:
I am a little unsure why we need this switch block. If we need it, it's much easier if we use enums to make it easier to differentiate between the cases.

Reply:
Removed switch block to an if/else block.